app/views/comments/create.html.haml

```diff
@@ -0,0 +1,2 @@
+%h1 Comments#create
+%p Find me in app/views/comments/create.html.haml
```

Comment:
don' need this file. because comments box is located at  show_page of prototypes. and comment system is made by using ajax.... so you don't need create page.